README.md

```diff
@@ -36,7 +40,10 @@ If you want to auto-configure your VVV machine with those add the following to y
 
 After that to run the tests for ACF 4 run `npm run test-acf4` inside the plugin folder.
 
-To run the tests for ACF 5 you first need to install ACF 5 as we cannot do this automatically as it isn't a free plugin.
-After you've done that you can simply run `npm run test-acf5pro`.
+To run tests for ACF 5 you first need to install ACF 5 manually, as we cannot do this automatically due to it not being a free plugin.
+After you've installed ACF 5, you can simply run `npm run test-acf5pro`.
+
+**Note:**
+If you do not want to run the browser tests with Chrome and/or in headless mode, you'll need to change `nightwatch.json`, `nightwatch.conf.example.js` and `env.js` in your virtual machine. 
```

Comment:
This was intentionally written using `some` and not `and` as depending on what you are trying to do only some need to be changed.

This is the relevant docs, maybe a link would be nice though: http://nightwatchjs.org/gettingstarted#settings-file